plugins/analysis-nori/src/main/java/org/elasticsearch/plugin/analysis/nori/NoriTokenizerFactory.java

```diff
@@ -56,7 +56,7 @@ public static UserDictionary getUserDictionary(Environment env, Settings setting
             USER_DICT_PATH_OPTION,
             USER_DICT_RULES_OPTION,
             DEDUPLICATE_DICTIONARY,
-            true,
+            false,  // typically don't want to remove comments as deduplication will provide better feedback
```

Comment:
why are we changing behavior here? We shouldn't change this behavior in this commit.

Reply:
The underlying UserDictionary in Lucene takes care of comments for us.  So not sure why the NoriTokenizerFactory had this set as true by default.  It was causing the deduplication logic to report incorrect line numbers (as getting the word list prior to deduplication is removing the comments).  

If there's a good reason to leave it as true I can and I can potentially try to update the deduplication logic checks to account for removed comments.  But this seemed reasonable to me as made that deduplication logic more helpful.